prime-dotnet-webapi/Controllers/SitesController.cs

```diff
@@ -497,12 +563,12 @@ public async Task<IActionResult> UpdatePecCode(int siteId, FromBodyText pecCode)
         /// Gets a download token for the latest business licence on a site.
         /// </summary>
         /// <param name="siteId"></param>
-        [HttpGet("{siteId}/latest-business-licence", Name = nameof(GetLatestBusinessLicenceDownloadToken))]
+        [HttpGet("{siteId}/business-licence/document-token", Name = nameof(GetBusinessLicenceDocumentToken))]
```

Comment:
Since we now have `/business-licence/document` for creating the document, this URL can be `business-licence/document/token`

Reply:
As well, the URL example in the comments for this method doesn't match the new URL.